lib/pbench/server/api/resources/query_apis/metadata_index/namespace_and_rows.py

```diff
@@ -249,9 +249,6 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
         dataset = context["dataset"]
         scroll_id = json_data.get("scroll_id")
         document = self.ES_INTERNAL_INDEX_NAMES.get(json_data["type"])
-        if not document:
-            self.logger.debug(f"Illegal document name {json_data['type']!r}")
-            abort(HTTPStatus.NOT_FOUND, message="Namespace not found")
 
         document_index = document["index"]
```

Comment:
Since you're no longer checking the value returned by `get()` to see if it is `None`, you should replace `get()` with `[]`-access:  that way, if something goes wrong, you get a nice, clear exception at the point of error, instead of getting a confusing exception from the next line.